tests/unit/api/test_token.py

```diff
@@ -77,12 +77,47 @@ def test_post(self, monkeypatch, test_client):
         token_data = {
             "username": "admin",
             "password": "secret",
-            "scope": "write:targets",
+            "scope": (
+                "read:bootstrap read:settings read:tasks read:token "
+                "write:targets delete:targets"
+            ),
```

Comment:
Here I added all allowed scopes to make sure we can create a token containing each of them and such bugs as #187 won't happen for other scopes unnoticed.